code/modules/mob/living/silicon/mommi/mommi_modules.dm

```diff
@@ -4,6 +4,11 @@
 
 /obj/item/weapon/robot_module/mommi/New(var/mob/living/silicon/robot/R)
 	..()
+	
+	//crabs are cute and therefor exempt from burning to death
+	heat_level_max = 1.#INF
```

Comment:
Real criticism: Use `INFINITY` and redefine `INFINITY` to `1.#INF` if you have to.